### ISSUE TITLE
redo MFCCs

### ISSUE THREAD

**Issue by superbock** [CPJKU/madmom#42]
Set the filterbank and transform to MelFilterbank and dct, respectively? There's always the Cepstrogram class if other parameters are needed / wanted.

Move the FFCC_* constants into the class.

**Comment by bharadwajdes** [comment]
Hi guys 
I would like to take up this task and try. please let me know if some is already working on this.
Thanks

**Comment by superbock** [comment]
As far as I know no one is working on this yet, so go ahead!

I assume you use MFCCs by yourself, so it would be great if you could also add tests to verify that the obtained values are correct.

**Comment by bharadwajdes** [comment]
Yeah sure I would write the tests as well before committing.  Thanks!
